### PR TITLE
[SQL] `az sql mi partner-cert`: New command group to manage partner certificates

### PR DESCRIPTION
**Pull request by ostojicb**
**Related command**

`az sql mi`

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
SQL

File: src/azure-cli/azure/cli/command_modules/sql/aaz/latest/sql/mi/partner_cert/_create.py [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "sql mi partner-cert create",
+)
+class Create(AAZCommand):
+    """Uploads a server trust certificate from box to Sql Managed Instance.
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/sql/aaz/latest/sql/mi/partner_cert/_create.py:18]:
```suggestion
    """Upload a server trust certificate from box to Sql Managed Instance.
```

File: src/azure-cli/azure/cli/command_modules/sql/aaz/latest/sql/mi/partner_cert/_show.py [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "sql mi partner-cert show",
+)
+class Show(AAZCommand):
+    """Gets a server trust certificate that was uploaded from box to Sql Managed Instance.
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/sql/aaz/latest/sql/mi/partner_cert/_show.py:18]:
```suggestion
    """Get a server trust certificate that was uploaded from box to Sql Managed Instance.
```

**Comment by ostojicb** [comment]
hi @evelyn-ys, this is ready for review/merge as well

**Review by evelyn-ys — APPROVED** [review]

**Pull request merged**